openmls_rust_crypto/Cargo.toml

```diff
@@ -26,8 +26,4 @@ rand_chacha = { version = "0.3" }
 hpke = { version = "0.1.0", package = "hpke-rs", default-features = false, features = ["hazmat", "serialization"] }
 hpke-rs-crypto = { version = "0.1.1" }
 hpke-rs-rust-crypto = { version = "0.1.1" }
-thiserror = "1.0"
-
-[patch.crates-io]
```

Comment:
They have been moved to the root `Cargo.toml` since anyway [they wouldn't be recognized in a workspace environment](https://doc.rust-lang.org/cargo/reference/workspaces.html#key-features)